src/utils/BlockchainUtils.js

```diff
@@ -0,0 +1,50 @@
+const ethers = require('ethers');
+
+class BlockchainUtils {
+
+  static getSignerOfMessage(message, signature) {
+    return ethers.utils.verifyMessage(message, signature);
+  }
+
+  static getAddressFromJSON(json) {
+    const address = ethers.utils.getJsonWalletAddress(json);
+    return BlockchainUtils.safeAdd0x(address);
+  }
+
+  static safeAdd0x(address) {
+    return address.startsWith('0x') ? address : `0x${address}`;
+  }
+
+  static safeAreAddressesEqual(addressOne, addressTwo) {
+    addressOne = BlockchainUtils.safeAdd0x(addressOne);
+    addressTwo = BlockchainUtils.safeAdd0x(addressTwo);
+    return addressOne.toLowerCase() === addressTwo.toLowerCase();
+  }
+
+  static isWalletInputValid(encryptedWallet, signature) {
+    const targetAddress = BlockchainUtils.getAddressFromJSON(encryptedWallet);
+    const signerOfMessage = BlockchainUtils.getSignerOfMessage(process.env.BLOCKCHAIN_SERVICE_SIGN_MESSAGE, signature);
```

Comment:
I think that the name of the variable might be misleading. `blockchain_service_sign_message` might be confused with something related the the nodejs api that we have "blockchain-service".

Just an idea: As this would be used in 2 places - client side and backend, can we use the env variable for the client side?